web/bulk_rendering/README.md

```diff
@@ -7,9 +7,11 @@ This renderer loads all the cloud keyboards from api.keyman.com and renders each
 1. `build.sh` to build the renderer (must have already built KeymanWeb in ../source/, `./build.sh -debug`).
 2. Open `index.html` (via http, not via file, as otherwise fonts will not be accessible).
 3. If you want a touch layout, use Developer Tools to select the appropriate emulation and then reload the page.
-3. Choose whether you wish to render all layers or just the default layer for each keyboard.
-4. If desired, filter out keyboards by id (regex).
+4. Choose whether you wish to render all layers or just the default layer for each keyboard.
+5. If desired, filter out keyboards by id (regex).
 5. Run the render.
+    - When prompted to screenshare, be sure to share _the Chrome tab_ if emulating mobile.  The screen capture
+      system will fail to capture the OSK properly otherwise.
```

Comment:
I'm not sure if this is helpful, because we don't want to ever run this in emulation anyway, so it might be misdirecting?

Reply:
Considering this was unrunnable on my personal iPhone - the screen capture prompt never displayed - but runnable in Chrome emulation of an iPhone, I found it helpful.  I relied on mobile-device emulation to get everything working again, and it took me a while to realize this was behind a blocker or two I had to deal with on the way.

Reply:
Also, it's worth noting the original point 3 above:

> If you want a touch layout, use Developer Tools to select the appropriate emulation and then reload the page.